### DISCUSSION TITLE
What's the difference between `view` and `panel`?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [microsoft/vscode-discussions#348] [Extension Development QnA] [answered]
Hi,

In VSCode there are two concepts, 
`view` and `panel`, what's the differences between them, in general people regard them as the same thing.

**Comment by liaodalin19903** [comment, marked as answer]
This image display the concepts:
![image](https://user-images.githubusercontent.com/12727582/210804996-d6cf36d0-ed41-462c-82ba-2b894066211f.png)

Panel is views' container.
Details:

[View](https://code.visualstudio.com/api/ux-guidelines/views) 

[Panel](https://code.visualstudio.com/api/ux-guidelines/panel)

**Reply by shoulders** [reply]
You answer and image are the ideal answer. So many people use the wrong names for things in VSCode. 

**i.e.** `Problems Panel`. It is not a `Panel`, it is a `View Container` in the `Panel area`. 😄
